crates/rpc/rpc-types/src/eth/transaction/receipt.rs

```diff
@@ -9,14 +9,16 @@ pub struct TransactionReceipt {
     /// Transaction Hash.
     pub transaction_hash: Option<H256>,
     /// Index within the block.
+    #[serde(skip_serializing_if = "Option::is_none")]
     pub transaction_index: Option<U256>,
```

Comment:
this should actually not be an Option at all.

so we should change this to U64 probably